app/controllers/admin/events/list.js

```diff
@@ -1,129 +1,178 @@
-      disableSorting : true
+import { computed, action } from '@ember/object';
+import { or } from '@ember/object/computed';
+import EmberTableControllerMixin from 'open-event-frontend/mixins/ember-table-controller';
+
+
+export default class extends Controller.extend(EmberTableControllerMixin) {
+
+   @or('authManager.currentUser.isSuperAdmin', 'authManager.currentUser.isAdmin') hasRestorePrivileges;
+
+   @computed()
+   get columns() {
+     return [
+       {
+         name            : 'Name',
+         valuePath       : 'name',
+         isSortable      : true,
+         headerComponent : 'tables/headers/sort',
+         cellComponent   : 'ui-table/cell/cell-event',
+         options         : {
+           hasRestorePrivileges: this.hasRestorePrivileges
+         },
+         actions: {
+           moveToDetails        : this.moveToDetails.bind(this),
+           editEvent            : this.editEvent.bind(this),
+           openDeleteEventModal : this.openDeleteEventModal.bind(this),
+           deleteEvent          : this.deleteEvent.bind(this),
+           restoreEvent         : this.restoreEvent.bind(this)
+         }
+       },
+       {
+         name          : 'Starts At',
+         valuePath     : 'startsAt',
+         cellComponent : 'ui-table/cell/cell-simple-date',
+         width         : 40,
+         options       : {
+           dateFormat: 'MMMM DD, YYYY - hh:mm A'
+         }
+       },
+       {
+         name          : 'Ends At',
+         valuePath     : 'endsAt',
+         cellComponent : 'ui-table/cell/cell-simple-date',
+         width         : 40,
+         options       : {
+           dateFormat: 'MMMM DD, YYYY - hh:mm A'
+         }
+       },
+       {
+         name      : 'State',
+         valuePath : 'state',
+         width     : 50
+       },
+       {
+         name          : 'Roles',
+         valuePath     : 'roles',
+         cellComponent : 'ui-table/cell/cell-roles'
+       },
+       {
+         name          : 'Sessions',
+         valuePath     : 'sessions',
+         cellComponent : 'ui-table/cell/cell-sessions',
+         width         : 60
+
+       },
+       {
+         name          : 'Speakers',
+         valuePath     : 'speakers',
+         cellComponent : 'ui-table/cell/cell-speakers-dashboard',
+         width         : 60
+       },
+       {
+         name          : 'Tickets',
+         valuePath     : 'tickets',
+         cellComponent : 'ui-table/cell/cell-tickets'
+       },
+       {
+         name          : 'Public URL',
+         valuePath     : 'url',
+         cellComponent : 'ui-table/cell/cell-link'
+       },
+       {
+         name          : 'Featured Event',
+         valuePath     : 'isFeatured',
+         cellComponent : 'ui-table/cell/admin/event-is-featured',
+         width         : 40,
+         actions       : {
+           toggleFeatured: this.toggleFeatured.bind(this)
+         }
+       }
+     ];
+   }
+
+  @computed('model.data.[]')
+   get rows() {
```

Comment:
Also, what is the purpose of this ? Can't you use the model data directly ?

Reply:
you mean instead of defininf rows, pass rows=model.data in the template ? @niranjan94

Reply:
Yep.